src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -0,0 +1,90 @@
+import React, { useCallback, useMemo, useState } from "react";
+import { Button, Nav } from "react-bootstrap";
+import { Link } from "react-router-dom";
+import classNames from "classnames";
+
+import AdvancedSettings from "pages/Admin/AdvancedSettings";
+import BasicInfo from "pages/Admin/BasicInfo";
+import EntranceExperience from "pages/Admin/EntranceExperience";
+import VenueDetails from "pages/Admin/Venue/Details";
+
+import { Venue_v2 } from "types/venues";
+
+import "./AdminVenueView.scss";
+
+export interface SidebarOptionType {
+  [key: string]: string;
+}
+
+export enum SidebarOptions {
```

Comment:
[NB] Enums should be non-plural:

```suggestion
export enum SidebarOption {
```

Reply:
Noted.